dynamiq/nodes/agents/base.py

```diff
@@ -57,7 +57,9 @@ class Agent(Node):
     llm: Node = Field(..., description="LLM used by the agent.")
     group: NodeGroup = NodeGroup.AGENTS
     error_handling: ErrorHandling = ErrorHandling(timeout_seconds=600)
-    streaming: StreamingConfig = StreamingConfig()
+    streaming_mode: StreamingMode = Field(
```

Comment:
I would keep it as `streaming`. Essentially, this is a still node's steaming configuration, but with a different set of parameters.

Reply:
Already updated this!

Reply:
Do we use the same event name for the final output and intermediate steps?